examples/todoapp/openapi.yaml

```diff
@@ -5,50 +5,110 @@ info:
   title: A simple Todo app API
 
 paths:
-
-  /:
-
+  /todos:
     get:
       summary: List of my TODO items
+      operationId: todo.list
+      parameters:
+        - $ref: "#/components/parameters/limit"
       responses:
-        '200':
+        "200":
           description: A list of my TODO items
           content:
             application/json:
               schema:
                 type: array
                 items:
                   $ref: "#/components/schemas/Item"
+        "400":
+          $ref: "#/components/responses/ValidationError"
+        "500":
+          $ref: "#/components/responses/ValidationError"
```

Comment:
500 errors are not validation errors. I see two options:
* remove them from the spec
* define them in the spec as something else than validation errors, I usually have a generic `ServerError` for this.

Reply:
I removed them.